mysql-test/suite/rocksdb/t/ddse_upgrade_row_lock.test

```diff
@@ -0,0 +1,33 @@
+--source include/have_rocksdb.inc
+--source include/have_innodb_ddse.inc
+
+
+--let $MYSQLD_DATADIR= `select @@datadir`
+
+--let $shutdown_server_timeout= 10
+--source include/shutdown_mysqld.inc
+
+# save data
+let $tmp_dir = $MYSQLTEST_VARDIR/tmp;
+let $saved_datadir = $tmp_dir/saved_datadir;
+force-cpdir $MYSQLD_DATADIR $saved_datadir;
+
+# There are 3 place during DD upgrade which may involves a lot row locks
+# - insert select statement: copy data from one storage engine to another storage engine, this only happens for DD SE upgrade, with this change, no row lock is hold
+# - drop DD tables: when drop DD table, SQL layer read each row and delete row. This happens for both DD SE change and DD version upgrade. max row lock is max number of rows in these DD tables
+# - SYSTEM table update: there are a lot sql scripts involves insert data into system table. This happens for both DD SE change and DD version upgrade. Currently some tables may involves 15-20K rows locks 
+--echo upgrade from INNODB->ROCKSDB DDSE
+let restart_parameters="restart: --default_dd_system_storage_engine=rocksdb --datadir=$saved_datadir --rocksdb_max_row_locks=20000";
```

Comment:
Why 20000? Can you go lower, i.e. 0 (if that means actual zero and not unlimited) or 1?

Reply:
currently no. due to system table upgrade(these sql script) still hold locks. may need another change to use bulk load for these sql scripts.